Source/ACE.Server/Managers/ServerManager.cs

```diff
@@ -69,6 +71,68 @@ public static void CancelShutdown()
             ShutdownInitiated = false;
         }
 
+        public static string CertificatePath
```

Comment:
To me these properties don't seem like they belong to ServerManager. The server can exist fine without them.

They seem more like TransferManager and WebApiManager type properties:

CertificatePath
TransferPath
SanitizedServerNameAsFileName
BasePath

Reply:
applicable functions moved to respective managers